app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html

```diff
@@ -22,7 +22,7 @@
 @import uk.gov.hmrc.nisp.controllers.auth.NispUser
 @import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
 
-@(nino: String, spSummary: SPSummaryModel, authenticationProvider: String, isPertaxUrl:Boolean
+@(nino: String, spSummary: SPSummaryModel, authenticationProvider: String, confidenceLevel: String, authProviderGA: Option[String], isPertaxUrl:Boolean
```

Comment:
Since the authprovider is an option we should be not just output that to the dimension. Because it will look like Some("verify") or None for example.
We should probably use something like `.map()` and only output the dimension if it exists